test/run.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-dir="$(dirname "$(realpath "$BASH_SOURCE")")"
+dir="$(dirname "$(readlink "$BASH_SOURCE")")"
```

Comment:
could we avoid hidden knowledge on mac os platforms?

Reply:
yes,  but it is additional custom code